.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb",
+  "rules": {
+    "indent": ["error", 2],
+    "max-len": ["error", 120],
+    "react/jsx-indent": ["error", 2],
+    "react/no-unused-prop-types": 0, // This gives a lot of false positives, lets disable for now
```

Comment:
Just saw your commnt on Slack
> About the eslint, have a look at https://github.com/andersinno/guide/blob/master/.eslintrc.js
> Let's use that, since our company standard.
Let me update the PR's code
Let